scripts/package-deploy.sh

```diff
@@ -46,6 +46,24 @@ function package {
     rm -rf $PACKAGE_STAGING_DIR
 }
 
+# package pulls built files for the platform ($1). If you pass in a file
+# extension ($2) it will be used on the binaries
+function package_zip {
```

Comment:
I prefer not to duplicate the logic from `package` function. I suggest passing a parameter that indicates the desired compression.

Reply:
Done